phalcon/session/adapter/redis.zep

```diff
@@ -118,7 +118,8 @@ class Redis extends Adapter
 	 */
 	public function read(sessionId) -> string
 	{
-		return (string) this->_redis->get(sessionId, this->_lifetime);
+	    let res = (string) this->_redis->get(sessionId, this->_lifetime);
+	    return res == null ? "" : res;
```

Comment:
You also need to instantiate the 'res' variable before this line:

    var res;